.github/workflows/all_jobs.yaml

```diff
@@ -1,4 +1,4 @@
-name: All Build on PR
+name: All Build

```

Comment:
What is the rationnal to remove "on PR"
It seems PR criteria is still in place below using "github.event_name == 'pull_request' "

Reply:
clarified by IM